src/dashboard/src/media/js/jobs.js

```diff
@@ -472,11 +471,13 @@ var MicroserviceGroupView = Backbone.View.extend({
 
         // Add link to browse SIP before it's made into an AIP
         if (
-          (job.get('currentstep') == job.sip.statuses['STATUS_AWAITING_DECISION'] && -1 < jQuery.inArray(job.get('type'), ['Store AIP', 'Approve normalization']))
-          || job.get('type') == 'Move to the uploadedDIPs directory'
-          || job.get('type') == 'Store DIP'
```

Comment:
`Store DIP` not included in my change because is unused - see https://github.com/archivematica/Issues/issues/62.